tools/deserialize-proto/main.go

```diff
@@ -3,34 +3,33 @@ package main
 import (
 	"bufio"
 	"encoding/hex"
-	"encoding/json"
 	"fmt"
 	"log"
 	"os"
+	"reflect"
 	"strconv"
 	"strings"
 
+	"github.com/golang/protobuf/jsonpb"
 	"github.com/golang/protobuf/proto"
 	flag "github.com/spf13/pflag"
+	_ "github.com/stackrox/rox/generated/storage"
```

Comment:
Let's add a comment that will explain why we need it (e.g. register all generated types)